packages/beacon-node/src/db/repositories/backfillState.ts

```diff
@@ -0,0 +1,29 @@
+import {ContainerType, ListUintNum64Type, ValueOf} from "@chainsafe/ssz";
+import {ChainForkConfig} from "@lodestar/config/lib";
+import {DatabaseController, Repository} from "@lodestar/db";
+import {Epoch, ssz} from "@lodestar/types";
+import {Bucket, getBucketNameByValue} from "../buckets.js";
+import {BACKFILL_RANGE_KEY} from "../single/backfillRange.js";
+
+// TODO: Fix the null possibilities. ssz types dont work with null values
+export const epochBackfillStateSSZ = new ContainerType(
+  {
+    hasBlock: ssz.Boolean,
+    hasBlobs: ssz.Boolean || null,
+    columnIndices: new ListUintNum64Type(128) || null,
```

Comment:
Maybe we can use a `ListBasicType` here.  What do you think?
```suggestion
    columnIndices: new ListBasicType(ssz.ColumnIndex, NUMBER_OF_COLUMNS),
```

Reply:
i thought `ListUintNum64Type` was more optimized, but recently saw that `ListBasicType` is used in most of the places, so changing to it.